### ISSUE TITLE
Armor Displays Armor Stand Do NOT Break When Display is Broken (1.17)

### ISSUE THREAD

**Issue by NazzyDragon** [Lightman314/LightmansCurrency#10]
Title. Place an armor display. Break it (creative and survival). Armor stand stays behind. Cannot be killed via /kill (even targetting all entities nearby not just stands). Lava. Etc. Do not conform to gravity. Kind of a big issue. Unsure if happens in singleplayer, is currently happening in multiplayer.

Modlist (I only know how to pull this from the console log, sorry if it's a bit messy):
https://pastebin.com/kVZBPqtA

**Comment by Lightman314** [comment]
Does the issue persist even after logging out and logging back in, cause it seems like this is a client desync issue (client probably wasn't informed that the armor stand was killed). I tested this on a server-client connection, and it seems to work as intended and I am thus unable to replicate the issue.

**Comment by NazzyDragon** [comment]
Apologies for not including the version in the title, figured every line in my pastebin listing 1.17 would be enough. Lol. I guess not. 

Killing *specifically* armor stands does work, really odd that killing all entities in a radius doesn't. Thanks fhe the response! This works for now, not like I plan to run a large server. Love this little currency mod, it's really nice. :) I've messed around with it a lot and this is the only issue I've found, so it's good so far!

**Comment by Lightman314** [comment]
I know that this issue has been open for a while, but I figured I'd give you an update while it's on my mind.
Now that Minecraft 1.18 has been released and I've finished porting my mod to 1.18, I'm probably only going to only do one last update for 1.17 to bring it up-to-date with the current state of the 1.16 & 1.18 versions. This update should fix this issue, and should hopefully be out before the weekend hits.
